packages/peregrine/lib/talons/ProductFullDetail/useProductFullDetail.js

```diff
@@ -200,3 +205,30 @@ export const useProductFullDetail = props => {
         quantity
     };
 };
+
+const getConfigPrice = (product, optionCodes, optionSelections) => {
+    let value = [];
```

Comment:
This can just be `let value;` as it will get set later. Also an array as a default seems wrong anyways.